### PR TITLE
Series permissions

### PR DESCRIPTION
**Pull request by martinfilliau**
WIP

See #204

### CODE REVIEW

**Comment by ahaith** [comment]
* [x] - Manageable editor set for a series
* [X] - Permission logic for series/event editing
* [ ] - Restrict and filter list of series in dashboard
* [ ] - Only show series in event edit which the user has permission to edit
* [ ] - Don't allow users to change or clear the series if they aren't editors of the series

File: talks/events/templatetags/users.py [hunk 1]
```diff
@@ -6,3 +6,7 @@
 @register.filter()
 def can_edit(user, event):
```
Comment (martinfilliau) [inline talks/events/templatetags/users.py]: Maybe this should be renamed `can_edit_event` to avoid future confusion?

**Comment by martinfilliau** [comment]
Looks good -- just to be sure, wasn't it planned that when you had the right to edit a series you were able to update all the talks belonging to that series?

File: talks/events/models.py [hunk 2]
```diff
@@ -365,11 +375,14 @@ def user_can_edit(self, user):
...
         They need to have the events.change_event permission AND be in the event's editors_set, or be a superuser
+        Users can also edit an event if they are an editor of the series it belongs to
         :param user: The django user wishing to edit the event
         :return: True if the user is allowed to edit this event, False otherwise
         """
-        return self.editor_set.filter(id=user.id).exists() or user.is_superuser
-
+        can_edit = self.editor_set.filter(id=user.id).exists() or user.is_superuser
+        if self.group:
+            can_edit = can_edit or self.group.user_can_edit(user)
```
Comment (ahaith) [inline talks/events/models.py]: @martinfilliau This line allows editors of a series to edit the talks within it

**Pull request merged**